bin/src/command/orders.rs

```diff
@@ -657,6 +662,7 @@ impl CommandServer {
                         }
                         ProxyResponseStatus::Error(message) => {
                             info!("return sockets error: {:?}", message);
+                            // we should return_error() here
```

Comment:
to me `break` and `return` here have the same effect

Reply:
actually `return_error()` is a distinct function that notifies the CommandServer about the error, and ultimately the client.